recipes/bytewax-feedstock/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "bytewax" %}
+{% set version = "0.16.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bytewax/bytewax/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 993fe70f3a206adc52aae1099c00e14a92ec55387dda4b0cda2b6f6fbf0ed35b
+  patches:
+    - ssl.patch
```

Comment:
Can you add a comment here explaining the patch and/or if it is upstream?

Reply:
done